features/demo/src/manifest.json

```diff
@@ -4,6 +4,8 @@
     "description": "Demonstrates some polyPod capabilities for testing purposes - please ignore",
     "primaryColor": "#000000",
     "thumbnail": "images/thumbnail.pdf",
+    "borderSize": "1",
```

Comment:
Will this really change feature by feature?

Reply:
mh good catch - I guess I went too much with this. 
the size it's fine to have it as a const that reflects the design 👍🏽  - thanks :)